pkg/client/client_test.go

```diff
@@ -33,9 +33,9 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-
 	kscheme "k8s.io/client-go/kubernetes/scheme"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
Only curious because it looks intentional and not the kind of thing I would expect `gofmt` to do. Why this change?

Reply:
Because we usually put imports form our own repo in a dedicated group, rather than mixing them with other imports in a common group